sshtunnel.py

```diff
@@ -59,13 +59,10 @@
 DEFAULT_LOGLEVEL = logging.ERROR  #: default level if no logger passed (ERROR)
 TRACE_LEVEL = 1
 logging.addLevelName(TRACE_LEVEL, 'TRACE')
+DEFAULT_SSH_DIRECTORY = '~/.ssh'
```

Comment:
maybe it's increases readability if we put the variable closer to `SSH_CONFIG_FILE` variable. To group SSH constants together.